sql-database/transparent-data-encryption/setup-tde-byok-sqlmi.ps1

```diff
@@ -4,23 +4,21 @@ Connect-AzAccount
 # If there are multiple subscriptions, choose the one where AKV is created: 
 Set-AzContext -SubscriptionId "subscription ID"
 
-# Install the preview version of AzureRM.Sql PowerShell package 4.11.6-preview if you are running this PowerShell locally (uncomment below):
-# Install-Module -Name AzureRM.Sql -RequiredVersion 4.11.6-preview -AllowPrerelease
+# Install the preview version of Az.Sql PowerShell package 1.1.1-preview if you are running this PowerShell locally (uncomment below):
+# Install-Module -Name Az.Sql -RequiredVersion 1.1.1-preview -AllowPrerelease -Force
 
 # 1. Create Resource and setup Azure Key Vault (skip if already done)
 
-# Create Resource (name the resource and specify the location)
+# Create Resource group (name the resource and specify the location)
 $location = "westus2" # specify the location
-New-AzResourceGroup -Name "MyRG" -Location $location
+New-AzResourceGroup -Name "MyRG" -Location $location # specify a new RG name
```

Comment:
The resource group name should be a variable, not called out in a comment. Otherwise the user has to do find/replace, rather than changing a single value.

Reply:
Done. Added variable to rest of the script

![image](https://user-images.githubusercontent.com/40007119/55820735-b2adf680-5ab0-11e9-903b-f3903cadcd62.png)